docs/4_2_co-simulation_api.adoc

```diff
@@ -115,6 +115,9 @@ Reasons for <<earlyReturn, `earlyReturn = fmi3True`>> are
 ** expressed by the return arguments <<eventHandlingNeeded>> or <<terminateSimulationDS>> of <<fmi3DoStep>>, or
 ** the FMU's reaction to the importers request for <<earlyReturn>> by the return argument <<earlyReturnRequested, `earlyReturnRequested = fmi3True`>> of <<fmi3IntermediateUpdateCallback>>, or
 ** the FMU's request for a communication point at <<lastSuccessfulTime>> for any other reason.
++ 
+<<earlyReturn,`earlyReturn = fmi3True`>> must not be used to signal that the requested step was not completed due to an error _[(e.g. integrator encountered division by zero)]_.
+Instead, the function must return with an error code (<<fmi3Error>> or <<fmi3Fatal>>).
```

Comment:
I think we also must point to fmi3Discard, which is another way the function can return immediately (without earlyReturn being true), e.g.:

```suggestion
Instead, the function must return with an error code (<<fmi3Error>> or <<fmi3Fatal>>).
Alternatively, the function can return with <<fmi3Discard>>, if the FMU is in the same state as before the call, i.e. the output argument values are undefined, but the computation may continue.
```